sample.env

```diff
@@ -9,3 +9,4 @@ REDIS_PORT = 6379
 REDIS_PASSWORD = 123
 REDIS_USERNAME = abc
 USE_REDIS_DB = true
+redisDefaultExpiryTime = 86400
```

Comment:
This Should be all caps. And default should be 1hr change to 24 hr from helm charts

Reply:
done